### DISCUSSION TITLE
Comparison with YOLO v4

### DISCUSSION THREAD

**Discussion by krcc5978** [Megvii-BaseDetection/YOLOX#1019] [Q&A]
Hi, 
I'd like to ask a question. 
Which program language did you use for comparison with Yolo v4?

I'm using Alexey AB's YOLO v4 now.
When I looked at your paper, YOLOX faster than YOLO v4.
But when I used YOLOX, YOLOX was about the same speed as YOLOX v4.
Maybe my measurements are wrong, but I thought there might be language differences.
So if you like, please tell me the language used in YOLO v4.

[enviroment]
・OS : windows10 Pro
・CPU: AMD Ryzen9 3900X
・GPU: RTX 3090 

PS.
Sorry for the poor English

**Comment by FateScript** [comment]
PTAL @Joker316701882 @GOATmessi7
